js/date-filter.js

```diff
@@ -12,6 +12,8 @@ require('jquery.inputmask/dist/inputmask/inputmask.numeric.extensions.js');
 function DateFilter(elm) {
   Filter.Filter.call(this, elm);
   this.validateInput = this.$elm.data('validate') || false;
+  this.$range = this.$elm.find('.range');
+  this.$grid = this.$elm.find('.date__range--grid');
```

Comment:
Ack, sorry, this is annoying, but can we tweak this to follow the BEM method more closely? `.block__element--modifier`. So in this case:

`.date-range__grid` where `.date-range` is the base block and `grid` is the sub-element, rather than a modifier.

Similarly: `.date-range__row`, `.date-range__year`, etc.